### PR TITLE
Add document subtype to the queue message

### PR DESCRIPTION
**Pull request by aliveni**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-720

### Change description ###
Add the "form type" to queue message.
"form type" value is the document subtype of the scannable item with document type as "FORM".

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/msg/EnvelopeMsg.java [hunk 1]
```diff
@@ -173,6 +178,16 @@ public String toString() {
             .filter(si -> si.getOcrData() != null);
     }
 
+    private String findSubtypeOfScannableItemWithFormType(Envelope envelope) {
```
Comment (nitinprabhu) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/msg/EnvelopeMsg.java]: should this be done after all OCR related validations has passed ?

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/msg/EnvelopeMsg.java [hunk 2]
```diff
@@ -173,6 +178,16 @@ public String toString() {
...
+        return envelope
+            .getScannableItems()
+            .stream()
+            .filter(si -> DocumentType.FORM.equals(si.getDocumentType()))
```
Comment (rkondratowicz) [inline src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/msg/EnvelopeMsg.java]: or `SSCS1` 😢

Comment (aliveni) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/msg/EnvelopeMsg.java]: `SSCS1` is the subtype value

Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/msg/EnvelopeMsg.java]: oh yeah, confused it with `InputaDocumentType`

File: src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/msg/EnvelopeMsg.java (hunk 1 above)
Comment (aliveni) [reply src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/msg/EnvelopeMsg.java]: @nitinprabhu yes, this is done after the ocr validations have passed and the envelope is processed
https://github.com/hmcts/bulk-scan-processor/blob/2682da5a7e5af553d9acb9784fb428de25401ca6/src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/OrchestratorNotificationTask.java#L59

**Pull request merged**